============================================================================================== 
Warning! Mixing Conda and module environments may lead to corruption of the
user environment. 
We do not recommend users mixing those two environments unless absolutely
necessary. Note that 
SURF does not provide any support for Conda environment.
For more information, please refer to our software policy page:
https://servicedesk.surf.nl/wiki/display/WIKI/Software+policy+Snellius#SoftwarepolicySnellius-UseofAnacondaandMinicondaenvironmentsonSnellius 

Remember that many packages have already been installed on the system and can
be loaded using 
the 'module load <package__name>' command. If you are uncertain if a package is
already available 
on the system, please use 'module avail' or 'module spider' to search for it.
============================================================================================== 
Already up to date.
Obtaining file:///gpfs/home6/jhuizing/master-thesis-ds/ptdraft
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Checking if build backend supports build_editable: started
  Checking if build backend supports build_editable: finished with status 'done'
  Getting requirements to build editable: started
  Getting requirements to build editable: finished with status 'done'
  Preparing editable metadata (pyproject.toml): started
  Preparing editable metadata (pyproject.toml): finished with status 'done'
Building wheels for collected packages: sisa3d
  Building editable for sisa3d (pyproject.toml): started
  Building editable for sisa3d (pyproject.toml): finished with status 'done'
  Created wheel for sisa3d: filename=sisa3d-0.1.0-py3-none-any.whl size=1286 sha256=466689da9623440a694956c93a4920038d607a9428abb993da0c7549d07f4ef9
  Stored in directory: /gpfs/scratch1/nodespecific/gcn13/jhuizing.6446630/pip-ephem-wheel-cache-vet9sjx8/wheels/ba/24/02/45c17a515af0a22326616b14da79c51e525e740ad6f0b007df
Successfully built sisa3d
Installing collected packages: sisa3d
  Attempting uninstall: sisa3d
    Found existing installation: sisa3d 0.1.0
    Uninstalling sisa3d-0.1.0:
      Successfully uninstalled sisa3d-0.1.0
Successfully installed sisa3d-0.1.0
[INFO] Creating temporary directory for config files: tmp_configs
Generated 1 configuration files in tmp_configs for 1 images.
[INFO] ✅ Temporary directory created with config files: tmp_configs
[INFO] Running sequantial grid search with 1 configuration files...
Running grid search:   0%|          | 0/1 [00:00<?, ?it/s][INFO] Running pipeline with config: tmp_configs/temporary_0_for_img_0.config.yaml
[INFO] Loading configuration...
[INFO] ✅ Configuration loaded.
[INFO] Loading paths...
[INFO] ✅ Paths loaded.
[INFO] Creating paths if they don't exist...
[INFO] ✅ Paths created.
[INFO] Initializing preprocessing pipeline...
[INFO] Running preprocessing pipeline...
[INFO] ✅ Preprocessing pipeline complete.
[INFO] Running DreamGaussian Stage 1 pipeline...
[INFO] ✅ DreamGaussian pipeline complete.
[INFO] Running Evaluation pipeline...
  0%|          | 0/2 [00:00<?, ?it/s] 50%|█████     | 1/2 [00:01<00:01,  1.87s/it]100%|██████████| 2/2 [00:01<00:00,  1.04it/s]
  0%|          | 0/2 [00:00<?, ?it/s] 50%|█████     | 1/2 [00:01<00:01,  1.01s/it]100%|██████████| 2/2 [00:01<00:00,  1.89it/s][INFO] ... Results saved to results/compare/compare.csv.
[INFO] ✅ Evaluation pipeline complete.
[INFO] Running DreamGaussian Stage 2 pipeline...
[ERROR] ❌ Subprocess error with non-zero exit status 1
[ERROR] Standard output of the subprocess: [INFO] ✅ DreamGaussian Configuration loaded.
[load_obj] use texture from: results/mesh/coarse/image-3
[load_obj] init empty albedo!
[Mesh loading] v: torch.Size([10558, 3]), f: torch.Size([17920, 3])
[Mesh loading] vn: torch.Size([10558, 3]), fn: torch.Size([17920, 3])
[INFO] load image from results/preprocessing/image-3 copy_rgba.png...

[ERROR] Standard error of the subprocess: Traceback (most recent call last):
  File "repos/dreamgaussian/main2.py", line 454, in <module>
    gui.train(opt.iters_refine)
  File "repos/dreamgaussian/main2.py", line 419, in train
    self.prepare_train()
  File "repos/dreamgaussian/main2.py", line 107, in prepare_train
    self.optimizer = torch.optim.Adam(self.renderer.get_params())
  File "/gpfs/home6/jhuizing/master-thesis-ds/repos/dreamgaussian/mesh_renderer.py", line 64, in get_params
    {'params': self.raw_albedo, 'lr': self.opt.texture_lr},
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 355, in __getattr__
    self._format_and_raise(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/base.py", line 231, in _format_and_raise
    format_and_raise(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/_utils.py", line 899, in format_and_raise
    _raise(ex, cause)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/_utils.py", line 797, in _raise
    raise ex.with_traceback(sys.exc_info()[2])  # set env var OC_CAUSE=1 for full trace
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 351, in __getattr__
    return self._get_impl(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 442, in _get_impl
    node = self._get_child(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/basecontainer.py", line 73, in _get_child
    child = self._get_node(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 480, in _get_node
    raise ConfigKeyError(f"Missing key {key!s}")
omegaconf.errors.ConfigAttributeError: Missing key texture_lr
    full_key: dreamgaussian.texture_lr
    object_type=dict


Traceback (most recent call last):
  File "pipeline.py", line 158, in <module>
    raise cpe
  File "pipeline.py", line 146, in <module>
    result = subprocess.run(command, check=True, text=True, capture_output=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['python', 'repos/dreamgaussian/main2.py', '--config', 'tmp_configs/temporary_0_for_img_0.config.yaml', 'input=results/preprocessing/image-3 copy_rgba.png', 'mesh=results/mesh/coarse/image-3 copy_rgba_mesh.obj', 'save_path=results/mesh/stage_2/']' returned non-zero exit status 1.
[ERROR] ❌ Subprocess error with non-zero exit status 1 for config: tmp_configs/temporary_0_for_img_0.config.yaml
Running grid search:   0%|          | 0/1 [01:24<?, ?it/s]
Traceback (most recent call last):
  File "grid_search.py", line 67, in <module>
    run_pipeline(config_path)
  File "grid_search.py", line 38, in run_pipeline
    raise cpe
  File "grid_search.py", line 30, in run_pipeline
    subprocess.run(["python", "pipeline.py", "--config", config_path], check=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['python', 'pipeline.py', '--config', 'tmp_configs/temporary_0_for_img_0.config.yaml']' returned non-zero exit status 1.
[INFO] Creating temporary directory for config files: tmp_configs
Generated 1 configuration files in tmp_configs for 1 images.
[INFO] ✅ Temporary directory created with config files: tmp_configs
[INFO] Running sequantial grid search with 1 configuration files...
Running grid search:   0%|          | 0/1 [00:00<?, ?it/s][INFO] Running pipeline with config: tmp_configs/temporary_0_for_img_0.config.yaml
[INFO] Loading configuration...
[INFO] ✅ Configuration loaded.
[INFO] Loading paths...
[INFO] ✅ Paths loaded.
[INFO] Creating paths if they don't exist...
[INFO] ✅ Paths created.
[INFO] Initializing preprocessing pipeline...
[INFO] Running preprocessing pipeline...
[INFO] ✅ Preprocessing pipeline complete.
[INFO] Running DreamGaussian Stage 1 pipeline...
[INFO] ✅ DreamGaussian pipeline complete.
[INFO] Running Evaluation pipeline...
  0%|          | 0/2 [00:00<?, ?it/s] 50%|█████     | 1/2 [00:01<00:01,  1.70s/it]100%|██████████| 2/2 [00:01<00:00,  1.15it/s]
  0%|          | 0/2 [00:00<?, ?it/s] 50%|█████     | 1/2 [00:00<00:00,  1.02it/s]100%|██████████| 2/2 [00:01<00:00,  1.94it/s][INFO] ... Results saved to results/compare/compare.csv.
[INFO] ✅ Evaluation pipeline complete.
[INFO] Running DreamGaussian Stage 2 pipeline...
[ERROR] ❌ Subprocess error with non-zero exit status 1
[ERROR] Standard output of the subprocess: [INFO] ✅ DreamGaussian Configuration loaded.
[load_obj] use texture from: results/mesh/coarse/image-3
[load_obj] init empty albedo!
[Mesh loading] v: torch.Size([11828, 3]), f: torch.Size([18906, 3])
[Mesh loading] vn: torch.Size([11828, 3]), fn: torch.Size([18906, 3])
[INFO] load image from results/preprocessing/image-3 copy_rgba.png...

[ERROR] Standard error of the subprocess: Traceback (most recent call last):
  File "repos/dreamgaussian/main2.py", line 454, in <module>
    gui.train(opt.iters_refine)
  File "repos/dreamgaussian/main2.py", line 419, in train
    self.prepare_train()
  File "repos/dreamgaussian/main2.py", line 107, in prepare_train
    self.optimizer = torch.optim.Adam(self.renderer.get_params())
  File "/gpfs/home6/jhuizing/master-thesis-ds/repos/dreamgaussian/mesh_renderer.py", line 64, in get_params
    {'params': self.raw_albedo, 'lr': self.opt.texture_lr},
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 355, in __getattr__
    self._format_and_raise(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/base.py", line 231, in _format_and_raise
    format_and_raise(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/_utils.py", line 899, in format_and_raise
    _raise(ex, cause)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/_utils.py", line 797, in _raise
    raise ex.with_traceback(sys.exc_info()[2])  # set env var OC_CAUSE=1 for full trace
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 351, in __getattr__
    return self._get_impl(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 442, in _get_impl
    node = self._get_child(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/basecontainer.py", line 73, in _get_child
    child = self._get_node(
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/site-packages/omegaconf/dictconfig.py", line 480, in _get_node
    raise ConfigKeyError(f"Missing key {key!s}")
omegaconf.errors.ConfigAttributeError: Missing key texture_lr
    full_key: dreamgaussian.texture_lr
    object_type=dict


Traceback (most recent call last):
  File "pipeline.py", line 158, in <module>
    raise cpe
  File "pipeline.py", line 146, in <module>
    result = subprocess.run(command, check=True, text=True, capture_output=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['python', 'repos/dreamgaussian/main2.py', '--config', 'tmp_configs/temporary_0_for_img_0.config.yaml', 'input=results/preprocessing/image-3 copy_rgba.png', 'mesh=results/mesh/coarse/image-3 copy_rgba_mesh.obj', 'save_path=results/mesh/stage_2/']' returned non-zero exit status 1.
[ERROR] ❌ Subprocess error with non-zero exit status 1 for config: tmp_configs/temporary_0_for_img_0.config.yaml
Running grid search:   0%|          | 0/1 [01:15<?, ?it/s]
Traceback (most recent call last):
  File "grid_search.py", line 67, in <module>
    run_pipeline(config_path)
  File "grid_search.py", line 38, in run_pipeline
    raise cpe
  File "grid_search.py", line 30, in run_pipeline
    subprocess.run(["python", "pipeline.py", "--config", config_path], check=True)
  File "/home/jhuizing/.conda/envs/dreamgaussianV2/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['python', 'pipeline.py', '--config', 'tmp_configs/temporary_0_for_img_0.config.yaml']' returned non-zero exit status 1.

JOB STATISTICS
==============
Job ID: 6446630
Cluster: snellius
User/Group: jhuizing/jhuizing
State: RUNNING
Nodes: 1
Cores per node: 18
CPU Utilized: 00:00:00
CPU Efficiency: 0.00% of 00:54:54 core-walltime
Job Wall-clock time: 00:03:03
Memory Utilized: 0.00 MB (estimated maximum)
Memory Efficiency: 0.00% of 120.00 GB (120.00 GB/node)
WARNING: Efficiency statistics may be misleading for RUNNING jobs.
